libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -362,7 +364,8 @@ newtype TeamCreationTime = TeamCreationTime
 
 data FeatureFlags = FeatureFlags
   { _flagSSO :: !FeatureSSO,
-    _flagLegalHold :: !FeatureLegalHold
+    _flagLegalHold :: !FeatureLegalHold,
+    _flagIndexedBillingTeamMembers :: !FeatureIndexedBillingTeamMembers
```

Comment:
This is really only required for an easier migration, right? Wondering if it would be easier to not make it a feature flag (it does not sound like sth we want to turn on/off) and also give it the default value of false, wdyt?

Reply:
> This is really only required for an easier migration, right? 

This is correct.

> Wondering if it would be easier to not make it a feature flag (it does not sound like sth we want to turn on/off) and also give it the default value of false, wdyt?

I am not sure what you mean by not feature flag? If you mean make it just a normal setting, I think it is the same effort. Giving it a default value might make our life easier with changes to our deployment setup and helm charts. But I've already made them.
Semantically, I agree it is a bit awkward, once it is turned on I don't see any reason anyone would want to turn it off. Let me know if you think it sits better with normal settings, it shouldn't be too much effort to move it.

Reply:
Yes, I think that because it's an option to help with migration, we should not make it a feature flag. And a default would help. Running that backfill can be in the docs but I don't see a good use to keep this around.

Also: we should have a way to check if the migration was done/needed to prevent someone from playing around with this option and putting galley in a bad state.

Reply:
Playing around with this option doesn't put galley in bad state. We write to the new table anyways. And checking if the migration was done would be fairly expensive unless we want to store a boolean in the database (we will have to seed it for new installations). Maybe it is ok for people to mess with this option, it would only mean billing could be broken for large teams, but data in C* will always be consistent. Also, we should probably remove this option in the next release (or the one after that) anyways

Reply:
>Playing around with this option doesn't put galley in bad state.

Ah, totally right - thanks for the reminder.

Reply:
To wrap up this topic: can we move this out of flags: the semantics of these flags is something like: they are something you turn on/off for specific teams, etc.

Let's move it then outside of the flags and give it a default plz?

Reply:
I have moved it.